### ISSUE TITLE
Request: Provide compiled XdocReport distribution zip for download on github

### ISSUE THREAD

**Issue by SuperPat45** [opensagres/xdocreport#186]
Hi,

Please provide compiled XdocReport distribution zip for download on github.

and It seem that there is no "xdocreport-2.0.0.jar" available on maven.

**Comment by pascalleclercq** [comment]
You are right. I'm a bit surprised. I don't understand why there is no xdocreport-2.0.0.jar in maven.
